config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits.yaml

```diff
@@ -171,45 +171,38 @@ presubmits:
       testgrid-tab-name: verify-boilerplate
       testgrid-alert-email: k8s-infra-staging-cluster-api-azure@kubernetes.io
   - name: pull-cluster-api-provider-azure-conformance-v1alpha3
+    path_alias: "sigs.k8s.io/cluster-api-provider-azure"
     always_run: false
     optional: true
     decorate: true
     decoration_config:
       timeout: 4h
+    max_concurrency: 5
     labels:
       preset-dind-enabled: "true"
       preset-kind-volume-mounts: "true"
-      preset-azure-cred: "true"
-    path_alias: "sigs.k8s.io/cluster-api-provider-azure"
-    extra_refs:
-    - org: kubernetes
-      repo: kubernetes
-      base_ref: v1.18.6
-      path_alias: k8s.io/kubernetes
+      preset-azure-cred-only: "true"
+      preset-azure-anonymous-pull: "true"
```

Comment:
do we need the ACR pull here?

Reply:
It pushes the images to the registry, do you think we don't need that?